asdfghjkl/symmatrix.py

```diff
@@ -223,13 +239,19 @@ def __add__(self, other):
             A=self.A.add(other.A), B=self.B.add(other.B), device=self.device
         )
 
+    def __iadd__(self, other):
+        self.A.add(other.A)
```

Comment:
This should be 
> self.A.add_(other.A) 

(in-place `add_` instead of `add`)

Can you confirm this (and for some other parts as well)